### PR TITLE
rework NIP-03 so it is actually usable

### PR DESCRIPTION
**Pull request by fiatjaf**

### CODE REVIEW

File: 03.md [hunk 1]
```diff
@@ -4,20 +4,22 @@ NIP-03
...
+```json
 {
-  "id": ...,
-  "kind": ...,
-  ...,
-  ...,
-  "ots": <base64-encoded OTS file data>
+  "kind": 1040
+  "tags": [
+    ["e", <relay-url>, <relay-url>],
```
Comment (pablof7z) [inline 03.md:15]: Event id

**Comment by pablof7z** [comment]
Should mention NIP-33 event attestation; maybe with an `a` tag for lookup and `e` tag of the attested version

**Comment by fiatjaf** [comment]
I think an `a` tag is not good here, since that may give people the wrong impression that the latest event of an `a` series is timestamped when actually only an old one is. If you have the `e` id you can fetch the actual event that was timestamped -- and if it was erased from all relays then the timestamp is useless anyway.

**Comment by pablof7z** [comment]
yeah, fair enough, I was just thinking for the lookup, but you're right that even the lookup case would be idiotic if you don't have the right `e`

**Review by staab — APPROVED** [review]

**Comment by vitorpamplona** [comment]
This is much better than the `ots` tag.

**Comment by jb55** [comment]
@fiatjaf it's not clear what exactly we're supposed to commit to in the ots file?

**Comment by jb55** [comment]
nm I can't read:

> The OpenTimestamps proof MUST prove the referenced `e` event id as its digest

**Pull request merged**